idb/postgres/postgres.go

```diff
@@ -2640,6 +2676,12 @@ func (db *IndexerDb) GetSpecialAccounts() (accounts idb.SpecialAccounts, err err
 	return
 }
 
+// DeleteAccount is used to remove an account from the database, needed for re-indexing an account. part of idb.IndexerDB.
+func (db *IndexerDb) DeleteAccount(address atypes.Address) error {
+	_, err := db.db.Exec(`DELETE FROM account WHERE account.addr = $1`, address[:])
```

Comment:
I think I need to delete `account_app` and `account_asset` as well.

Reply:
Thinking about this more, I think the RoundUpdate filter may need to exclude things that modify `asset` and `app`.

`asset` because there may be a different manager address, and including those transactions would be difficult.
`app` I'm not so sure about.

Reply:
Now I slightly wish I'd set up FOREIGN_KEY constraints in the schema and we could do delete with CASCADE.
Or maybe this is always a hackish tool that should never actually be used in prod? If that's the case maybe split it out to a separate cmd/reindex/